scripts/globals/battlefield.lua

```diff
@@ -82,6 +82,7 @@ xi.battlefield.id =
     NE_APOLLYON = 1292,
     SE_APOLLYON = 1293,
     CS_APOLLYON = 1294,
+    CENTRAL_APOLLYON = 1296,
```

Comment:
CI didn't trip for alignment here? interesting.

Reply:
Would be something nice to have added in the future, but I think that check is only on the cpp side for now.

Reply:
I can fix that alignment in just a few

Reply:
Actually, would you guys be opposed if I just filled out the entire thing with all existing battlefields? I can easily do this and would be much easier than adding them one at a time.

Reply:
works for me